SynthSeg/training_with_tfrecords.py

```diff
@@ -30,6 +37,8 @@ def training(opts: TrainingOptions) -> tf.keras.callbacks.History:
         opts: The training options. The parameters related to the generation of the synthetic images will be ignored.
     """
     # Check epochs
+    set_seed()
```

Comment:
I would reuse this keras method which does the same: https://www.tensorflow.org/api_docs/python/tf/keras/utils/set_random_seed

Reply:
ahh nice catch, replaced with keras method